src/view/components/countries/SelectedCountries.tsx

```diff
@@ -19,14 +19,15 @@ export const SelectedCountries = ({
     const [isPercentageError, setIsPercentageError] = React.useState(false);
 
     useEffect(() => {
-        const totalPercentage = Object.values(selectedCountries).reduce(
-            (total, percentage) => total + percentage
+        const totalPercentage = Array.from(selectedCountries.values()).reduce(
+            (total, entry) => total + entry,
+            0
         );
         setWorldPercentage(1 - totalPercentage);
     }, [selectedCountries]);
 
     useEffect(() => {
-        const hasPercentageError = Object.values(selectedCountries).some(
+        const hasPercentageError = Array.from(selectedCountries.values()).some(
```

Comment:
What's the difference between `Array.from(foo.values())` and `Object.values(foo)`?

Reply:
I have not exactly understood why, but Object.values() returns an empty array. It seems that map objects do not have an enumerable property and thus cannot be processed by Object.values.